src/web/head.hbs

```diff
@@ -23,4 +23,5 @@
   async
   defer
 ></script>
+<script src="https://cdn.jsdelivr.net/npm/algoliasearch@4.5.1/dist/algoliasearch-lite.umd.js"></script>
```

Comment:
this should be defer async to not block loading

Reply:
Ah good catch, fixed